frontend/src/components/user/loginPage.vue

```diff
@@ -1,9 +1,181 @@
 <template>
-  <div>login</div>
+  <div class="login-container">
+    <h1 class="title">Logowanie</h1>
+
+    <br>
+
+    <form @submit.prevent="submitLoginForm">
+      <b-field label="Login">
+          <b-input v-model="login"
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Hasło">
+          <b-input type="password" v-model="password"
+              password-reveal
+              minlength="8"
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <div class="notification is-danger" v-if="loginBtnErrors.length">
+        <p v-for="error in loginBtnErrors" v-bind:key="error">{{ error }}</p>
+      </div>
+
+      <br>
+
+      <button id="btnLogin" class="button">ZALOGUJ SIĘ</button>
+
+    </form>
+
+    <br>
+    <br>
+    <br>
+    <button id="btnRemindPassword" class="button" v-on:click="moveToRemindPassword">Nie pamiętam hasła.</button>
+
+    <br>
+    <br>
+
+    <div class="notification is-danger" v-if="remindPasswordError !== ''">
+      <p>{{ remindPasswordError }}</p>
+    </div>
```

Comment:
Czy to jest potrzebne? I tak podajemy tylko e-mail jak zapomnimy hasła

Reply:
Tu chodzi o to że może wystąpić błąd związany z podaniem nieistniejącego w bazie loginu i trzeba to gdzieś wyświetlić